config/mumbai.json

```
@@ -0,0 +1,22 @@
+{
+    "network": "mumbai",
+    "erc721": [
+        {
+            "name": "AavegotchiDiamond",
+            "address": "0x83e73D9CF22dFc3A767EA1cE0611F7f50306622e",
+            "startBlock": 34467860
+        },
+        {
+            "name": "RealmDiamond",
```

Let's always use the name of the game in the name of the NFT
```suggestion
            "name": "AavegotchiParcel",
```